src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSplitPaneDivider.java

```diff
@@ -394,7 +394,9 @@ public void paint(Graphics g) {
 
     /**
      * Messaged when the oneTouchExpandable value of the JSplitPane the
-     * divider is contained in changes. Will create the
+     * divider is contained in changes.
+     * If a particular L&amp;F supports SplitPane.supportsOneTocuhButtons property
+     * it will create the
```

Comment:
Tocuh ?

Reply:
DId you want "touch" instead of "create"?
I guess it should be "create" as the code does```

/* Create the left button and add an action listener to
expand/collapse it. */
leftButton = createLeftOneTouchButton();

Reply:
I was pointing out a spelling mistake. Nothing more.

Reply:
ok..fixed..

Reply:
Look like is not fixed yet?

Reply:
Spelling mistake "Tocuh" is fixed...Do you see more?